### PR TITLE
WFCORE-1347 - fix properties attribute to allow null as it is expecit…

### PR DESCRIPTION
**Pull request by ctomc**
…ing it anyway

needed for https://github.com/wildfly/wildfly-core/pull/1400

### CODE REVIEW

**Comment by ctomc** [comment]
Integration job with both PRs https://ci.wildfly.org/viewLog.html?buildId=9239&tab=buildResultsDiv&buildTypeId=WF_WildFlyCoreIntegrationExperiments

**Comment by ctomc** [comment]
retest this please

**Comment by ctomc** [comment]
integration job passes https://ci.wildfly.org/viewLog.html?buildId=9266&tab=buildResultsDiv&buildTypeId=WF_WildFlyCoreIntegrationExperiments

File: testsuite/integration/src/test/scripts/clustering-common-targets.xml [hunk 1]
```diff
@@ -197,6 +197,24 @@
         </sequential>
     </macrodef>
 
+
+    <!--
+          Add a new cache container to the Infinispan subsystem.
+          The container definition is loaded from a file.
+      -->
```
Comment (bstansberry) [inline testsuite/integration/src/test/scripts/clustering-common-targets.xml]: Looks like a cut-and-paste comment.

File: testsuite/integration/src/test/scripts/clustering-common-targets.xml [hunk 2]
```diff
@@ -197,6 +197,24 @@
...
+      <macrodef name="ts.config-as.enable-trace-http-method" description="Enable trace http method in undertow subsystem">
+
+          <attribute name="name" default="wildfly"/>
+          <attribute name="output.dir" default="${project.build.directory}"/>
+          <attribute name="config.dir.name" default="standalone/configuration"/>
+
+          <sequential>
+              <echo message="Adding a cache container to Infinispan subsystem for config @{name}" />
```
Comment (bstansberry) [inline testsuite/integration/src/test/scripts/clustering-common-targets.xml]: Looks like cut-and-paste

File: testsuite/integration/src/test/xslt/enableHttpTrace.xsl [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   <subsystem xmlns="urn:jboss:domain:undertow:3.1">
+        <buffer-cache name="default" />
+        <server name="default-server">
+            <http-listener name="default" socket-binding="http" redirect-socket="https" />
+            ...
+        </server>
+        ..
+    -->
+
+    <!-- Add the interface with configured inetAddress -->
```
Comment (bstansberry) [inline testsuite/integration/src/test/xslt/enableHttpTrace.xsl]: The comments from L26 to here don't seem related.

**Comment by ctomc** [comment]
fixed
